src/signalo/edge_cases/tests.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.test import APITestCase
+
+
+class TestViewsets(APITestCase):
+    def test_allowanymodel(self):
+        response = self.client.get(
+            "/oapif/collections/signalo_edge_cases.allowanymodel"
+        )
+        self.assertIs(response.status_code, 200)
+
+    def test_isadminusermodel(self):
+        response = self.client.put(
+            "/oapif/collections/signalo_edge_cases.isadminusermodel", data={}
+        )
+        self.assertIn(response.status_code, [403, 405])
```

Comment:
Here I think you could test all four CRUD operations, with three type of requests (non authenticated, authenticated without permissions, and authenticated with permissions), each time asserting either success and failure.

It's likely a bit redudent to DRF's tests, but since it's a very sensitive topic I think it's worht it